src/doc/trpl/strings.md

```diff
@@ -24,9 +24,29 @@ compiled program, and exists for the entire duration it runs. The `greeting`
 binding is a reference to this statically allocated string. String slices
 have a fixed size, and cannot be mutated.
 
-A `String`, on the other hand, is a heap-allocated string. This string is
-growable, and is also guaranteed to be UTF-8. `String`s are commonly created by
-converting from a string slice using the `to_string` method.
+String literals can span multiple lines. There are two forms. The first will
+include the newline and the leading spaces:
+
+```rust
+let s = "foo
+    bar";
+
+assert_eq!("foo\n        bar", s);
+```
+
+The second, with a `\`, does not:
```

Comment:
Perhaps this could go into a little more detail about what's going on here? It basically just means the newline and any leading whitespace on the next line is trimmed, but may be more clear if called out explicitly.